packages/web-pkg/src/components/FilesList/ResourceTiles.vue

```diff
@@ -297,7 +311,23 @@ export default defineComponent({
     }
 
     const isResourceClickable = (resource: Resource) => {
-      return !(unref(isEmbedModeEnabled) && !resource.isFolder)
+      return !(unref(isEmbedModeEnabled) && !unref(isFilePicker) && !resource.isFolder)
```

Comment:
This is giving me headache 😂 Any chance to simplify this? According to Chat-GPT this is the same, but I kinda doubt that tbh:

```suggestion
      return resource.isFolder || unref(isEmbedModeEnabled) || unref(isFilePicker)
```

Reply:
thats not the same equation oO 
anyways, I simplified it for ya